lib/l10n/intl_en.arb

```
@@ -293,6 +293,7 @@
   "disputeTradeDialogContent": "You are about to start a dispute with your counterparty. Do you want to continue?",
   "disputeCreatedSuccessfully": "Dispute created successfully",
   "disputeCreationFailed": "Failed to create dispute",
+  "disputeCreationError": "Error creating dispute",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)